bucket_sort/src/test/java/ru/nms/labs/BucketSortTest.java

```diff
@@ -0,0 +1,24 @@
+package ru.nms.labs;
+
+import org.junit.jupiter.api.Test;
+import ru.nms.labs.service.BucketSort;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class BucketSortTest {
+
+    private final BucketSort bucketSort = new BucketSort();
+
+    @Test
+    void shouldSortElements() {
+        List<Integer> input = TestDataGenerator.generateRandomList(1000, 0);
+        List<Integer> sortedInput = new ArrayList<>(input);
+        Collections.sort(sortedInput);
+        assertEquals(sortedInput, bucketSort.sort(input));
```

Comment:
На тему полезностей, советую глянуть на AssertJ, а именно [`isSorted()`][1]:

```suggestion
        assertThat(bucketSort.sort(input))
            .isSorted();
```

[1]: https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractListAssert.html#isSorted()